src/libs/API.js

```diff
@@ -179,8 +179,8 @@ Network.registerResponseHandler((queuedRequest, response) => {
         return;
     }
 
-    if (response.jsonCode === 405 || response.jsonCode === 404) {
-        // IOU Split & Request money transactions failed due to invalid amount(405) or unable to split(404)
+    if (response.jsonCode === 405 || response.jsonCode === 404 || response.jsonCode === 402) {
```

Comment:
@Santhosh-Sellavel We must stop rejecting the promise here. ANY API command that returns this error code will lead to an unhandled promise rejection unless it is caught...

Reply:
Fixing it in this PR https://github.com/Expensify/App/pull/8087 already but got conflicts because the bad practice continues 😅

Reply:
Sorry, I thought they were specific in this IOU Flow.  Will keep a note on this @marcaaron.